extensions/policies/src/main/java/org/eclipse/dataspaceconnector/mvd/SeedPoliciesExtension.java

```diff
@@ -55,8 +55,8 @@ public String name() {
      */
     @Override
     public void initialize(ServiceExtensionContext context) {
-        ruleBindingRegistry.bind("USE", ContractDefinitionService.NEGOTIATION_SCOPE);
-        ruleBindingRegistry.bind(ABS_SPATIAL_POSITION, ContractDefinitionService.NEGOTIATION_SCOPE);
+        ruleBindingRegistry.bind("USE", ALL_SCOPES);
+        ruleBindingRegistry.bind(ABS_SPATIAL_POSITION, ContractDefinitionService.CATALOGING_SCOPE);
```

Comment:
NEGOTIATION_SCOPE does not exists anymore?

Reply:
EDC upstream changes (this is documented in an EDC readme file, amended PR description to point to it)